### DISCUSSION TITLE
这种吸顶整块白色是要设置什么吗

### DISCUSSION THREAD

**Discussion by komorebi-cqd** [ant-design/pro-components#8923] [Q&A]
![微信图片_20241213142700](https://github.com/user-attachments/assets/078bf57b-f771-4d0e-88c3-45667cb1aae0)

官网示例（https://procomponents.ant.design/components/layout?tab=api#%E5%90%B8%E9%A1%B6-header）的这种吸顶效果是要设置什么吗，我看示例也就添加了fixedHeader，为什么我添加在我的项目中要滑动才会整块白色呢
![4](https://github.com/user-attachments/assets/ed611ca1-22a5-41c3-9efb-894630f1bd61)

![2](https://github.com/user-attachments/assets/b9b81543-c9a3-4904-b6c5-5a03d240aa6e)
![3](https://github.com/user-attachments/assets/7bc8cdbb-349e-4892-a74e-de872c532d91)

**Comment by komorebi-cqd** [comment]
@ant-design/pro-components的版本是2.3.57

**Comment by echoyl** [comment]
```ts
<PageContainer affixProps={{offsetTop: 0,style: { marginTop: -1 },}}></PageContainer>
```
affixProps 试试这个属性
